addons/scratchr2/forums.css

```diff
@@ -375,3 +375,7 @@ body > #pagewrapper {
   color: white;
   line-height: 32px;
 }
+
+.ocular-status {
+  word-wrap: break-word;
```

Comment:
The standard name for this property is `overflow-wrap`. It works in all browser versions supported by SA.

Reply:
Oops,